client/v3/config.go

```diff
@@ -90,3 +93,103 @@ type Config struct {
 
 	// TODO: support custom balancer picker
 }
+
+type ClientConfig struct {
```

Comment:
Note, let's avoid pointers in configuration. There is no benefit of using pointers here.

Reply:
I just moved the struct from etcdctl into clientv3.  I did not & do not want to change anything else so as not to impact the existing logic. If we want to change anything else, we'd better to do it in a separate PR. WDYT?

Reply:
ack